src/everything.rs

```diff
@@ -26,9 +27,9 @@ use crate::data::buildings::Building;
 use crate::data::casusbelli::{CasusBelli, CasusBelliGroup};
 use crate::data::character_templates::CharacterTemplate;
 use crate::data::characters::Characters;
-use crate::data::coa::{CoaDynamicDefinition, CoaTemplateList, Coas};
+use crate::data::coa::{CoaDynamicDefinition, Coas, CoaTemplateList};
 use crate::data::coadesigner::{
-    CoaDesignerColorPalette, CoaDesignerColoredEmblem, CoaDesignerEmblemLayout, CoaDesignerPattern,
+    CoaDesignerColoredEmblem, CoaDesignerColorPalette, CoaDesignerEmblemLayout, CoaDesignerPattern,
```

Comment:
Are you using different cargo fmt settings? I'm using the defaults (I think).

Reply:
Yea, I'm using the default settings, I think. It looks like it's just sorting it alphabetically. Does yours not do that? Maybe I'm on a different version.